src/promptflow/promptflow/_cli/_params.py

```diff
@@ -306,4 +306,12 @@ def add_param_verbose(parser):
     )
 
 
+def add_param_connection_provider(parser):
+    parser.add_argument(
+        "--connection-provider",
+        type=str,
+        help=argparse.SUPPRESS,
```

Comment:
do we expect end user to type this in their command?  if yes, this should not be suppressed

Reply:
No, just for ai studio for now.